tests/integration_tests/data_steward/cdr_cleaner/cleaning_rules/drop_invalid_sleep_level_records_test.py

```diff
@@ -0,0 +1,129 @@
+"""
+Integration test for drop_invalid_sleep_level_records module
+
+Rule to sandbox and drop records in the sleep_level table where level is
+not one of the following: awake, light, asleep, deep, restless, wake, rem, unknown.
+
+Original Issues: DC-2605
+"""
+
+# Python Imports
+import os
+from datetime import date, datetime
+
+# Project Imports
+from app_identity import PROJECT_ID
+from common import JINJA_ENV, SLEEP_LEVEL
+from cdr_cleaner.cleaning_rules.drop_invalid_sleep_level_records import DropInvalidSleepLevelRecords
+from tests.integration_tests.data_steward.cdr_cleaner.cleaning_rules.bigquery_tests_base import BaseTest
+
+SLEEP_LEVEL_TEMPLATE = JINJA_ENV.from_string("""
+INSERT INTO `{{project_id}}.{{dataset_id}}.{{sleep_level_table}}`
+(person_id, sleep_date, is_main_sleep, level, start_datetime, duration_in_min)
+VALUES
+(1, '2008-11-18','false', 'wake', '2008-11-18T00:00:00', 4.5),
+(2, '2010-01-01','true', 'light', '2010-01-01T00:00:00', 3.5),
+(3, '2012-01-01','true', 'invalid', '2010-01-01T00:00:00', 2.5),
+(4, '2010-11-18','false', 'rem', '2008-11-18T05:00:00', 7.5),
+(5, '2014-01-01','false', 'deep', '2014-01-01T05:00:00', 8.5),
+(6, '2013-03-14','true', 'awake', '2013-03-14T05:00:00', 6.6),
+(7, '2009-06-23','false', NULL, '2009-06-23T05:00:00', 3.8),
+(8, '2011-09-12','true', 'restless', '2011-09-12T05:00:00', 5.3),
+(9, '2015-01-21','true', 'unknown', '2015-01-21T05:00:00', 7.3),
+(10, '2017-02-05','false', 'asleep', '2017-02-05T05:00:00', 3.8),
+(11, '2010-07-18','false', 'invalid', '2010-07-18T05:00:00', 7.5)
+ """)
```

Comment:
This is good test data, @lledezma - could you add at least one data point that is a duplicate person_id to test out the scenario of some good data/some bad data?